src/components/Card/Card.module.css

```
@@ -57,17 +57,17 @@
  */
 .card--inverted {
   color: var(--eds-theme-color-text-neutral-default-inverse);
-  background: transparent;
+  background: transparent; /* 1 */
```

We would like the comment text down here instead of the number.
```suggestion
  /* Used on dark backgrounds to lighten the text color and remove a box shadow */
  background: transparent;
```

But something I just remembered is that we're actually going to remove `inverted` variants because none of those are being used and we have no plans to use them (again, those came with the basic repo we were given; our team didn't add these), so feel free to skip over anything with `inverted` in the name. (Here's the ticket: https://app.shortcut.com/czi-edu/story/213036/starter-task-remove-inverted-variant-in-eds-where-not-used)

---

Thanks for letting me know !